src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -276,7 +275,6 @@
     <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\WindowsRuntime\CLRIKeyValuePairImpl.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(BclSourcesRoot)\System\AggregateException.cs" />
```

Comment:
@ViktorHofer it seems the tests don't store serialized blobs for Exception and AggregateException specifically. If that's the case it would not detect issues serializing them between runtimes, I think? Do you recall why that is? 
```c#
        public static IEnumerable<object> SerializableExceptions_MemberData()
        {
            var exception = new Exception("Exception message", new Exception("Inner exception message"));
            yield return new object[] { new AggregateException("Aggregate exception message", exception) };
            yield return new object[] { exception };
        }
```